specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+            "modelAsString": true
+          }
+        },
+        "requestDate": {
+          "description": "Gets request date",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "justification": {
+          "description": "Gets or sets user's justification for the plan's request",
+          "type": "string"
+        },
+        "subscriptionId": {
+          "description": "Gets or sets the subscription id that the user is requesting to add the plan to",
+          "type": "string"
+        },
+        "subscriptionName": {
+          "description": "Gets or sets the subscription name that the user is requesting to add the plan to",
+          "type": "string"
+        }
+      }
+    },
+    "QueryRequestApproval": {
+      "description": "Gets the request plans with indication on each plan whether is approved by the admin, has pending request or not requested yet",
+      "type": "object",
+      "properties": {
+        "uniqueOfferId": {
+          "description": "Gets or sets unique offer id.",
+          "type": "string"
+        },
+        "plansDetails": {
+          "description": "Gets or sets the plans details",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/planDetails"
+          }
+        },
+        "etag": {
+          "description": "Gets or sets e-tag field ",
+          "type": "string"
+        },
+        "messageCode": {
+          "description": "Gets or sets the notification message id",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "QueryRequestApprovalProperties": {
+      "description": "The details to get the request plans statuses",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RequestDetails",
+          "description": "The details to get the request plans statuses"
+        }
+      }
+    },
+    "RequestDetails": {
+      "description": "Request details needed to get the plans statuses",
+      "type": "object",
+      "properties": {
+        "publisherId": {
+          "description": "The offer's publisher id",
+          "type": "string"
+        },
+        "planIds": {
+          "description": "Current plans list",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "subscriptionId": {
+          "description": "Gets or sets the subscription id",
+          "type": "string"
+        }
+      }
+    },
+    "NewPlansNotificationsList": {
+      "description": "List of all new plans notifications for public offers",
+      "type": "object",
+      "properties": {
+        "newPlansNotifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NewNotifications"
+          }
+        }
+      }
+    },
+    "SubscriptionsContextList": {
+      "description": "Get private store subscriptions context",
```

Comment:
>Get [](http://example.com/codeflow?start=22&length=3)

This should describe the contents of the response, not a method or operation. Maybe something more like "List of subscription Ids in the private store"?

Reply:
fixed

Reply:
There are many other instances in this file. I only commented on a couple. You should fix them all.

Reply:
I fixed the descriptions and properties names I found relevant.